browser/main/lib/modal.js

```diff
@@ -15,9 +15,13 @@ class ModalBase extends React.Component {
 
   close () {
     if (modalBase != null) modalBase.setState({component: null, componentProps: null, isHidden: true})
-    // Toggle overflow style on NoteList
-    let list = document.querySelector('.NoteList__list___browser-main-NoteList-')
-    list.style.overflow = 'auto'
+
+    // Toggle display for scrollbars
+    let list = document.querySelectorAll('.NoteList__list___browser-main-NoteList-, .SnippetNoteDetail__description___browser-main-Detail- > textarea')
```

Comment:
I agree. I had just take a clue from this [commit](https://github.com/BoostIO/Boostnote/commit/549c289f81964f7b6938cdd0a108e2253c86d263 ) on fixing the bug.